homeassistant/components/deconz/config_flow.py

```diff
@@ -271,15 +270,15 @@ async def async_step_deconz_devices(self, user_input=None):
                 {
                     vol.Optional(
                         CONF_ALLOW_CLIP_SENSOR,
-                        default=self.config_entry.options.get(
-                            CONF_ALLOW_CLIP_SENSOR, DEFAULT_ALLOW_CLIP_SENSOR
-                        ),
+                        default=self.gateway.option_allow_clip_sensor,
                     ): bool,
                     vol.Optional(
                         CONF_ALLOW_DECONZ_GROUPS,
-                        default=self.config_entry.options.get(
-                            CONF_ALLOW_DECONZ_GROUPS, DEFAULT_ALLOW_DECONZ_GROUPS
-                        ),
+                        default=self.gateway.option_allow_deconz_groups,
+                    ): bool,
+                    vol.Optional(
+                        CONF_ALLOW_NEW_DEVICES,
```

Comment:
What's the difference between this and the system option to disable new entities?

Reply:
The Main difference is the this doesn't create entries in the entity registry. Which is the requested behaviour in this case